CONTRIBUTING.md

```diff
@@ -3,6 +3,37 @@
 We'd love your help! Please join our weekly [SIG
 meeting](https://github.com/open-telemetry/community#special-interest-groups).
 
+## Target audiences
+
+The OpenTelemetry Collector has two main target audiences:
+
+1. End-users, aiming to use an OpenTelemetry Collector binary as a middleware on their infra, sitting between their
```

Comment:
I would remove the "usage example". Want to make sure that we don't limit to that usage :) Or say that as an example "a middleware ..."

Reply:
Do we need to refer to "middleware"? Just saying "use binary" should be enough, I think. 
```suggestion
1. End-users, aiming to use an OpenTelemetry Collector binary.
```

Reply:
Done